### PR TITLE
callcontext and copy table synchronization

### PR DESCRIPTION
**Pull request by ashWhiteHat**
Close https://github.com/privacy-scaling-explorations/zkevm-specs/issues/377
Part of https://github.com/privacy-scaling-explorations/zkevm-specs/issues/364

I updated CopyTable description and class fields.
I also gave serial numbers to callcontext constraints.

I would appreciate it if you could confirm.
Thank you!

### CODE REVIEW

File: src/zkevm_specs/state_circuit.py [hunk 1]
```diff
@@ -282,18 +282,24 @@ def check_call_context(row: Row, row_prev: Row):
...
 
-    # 5.0. Unused keys are 0
+    # 5.1. Unused keys are 0
     assert row.address() == 0
     assert row.storage_key() == 0
 
-    # 5.1 state root does not change
-    assert row.root == row_prev.root
+    # 5.2. field_tag is in CallContexFieldTag range
+    assert True
```
Comment (han0110) [inline src/zkevm_specs/state_circuit.py:290]: Is this a todo?

**Review by han0110 — COMMENTED** [review]
LGTM! Just a small question

File: src/zkevm_specs/state_circuit.py (hunk 1 above)
Comment (ashWhiteHat) [reply src/zkevm_specs/state_circuit.py]: It's my mistake.
I implemented it!

**Review by han0110 — APPROVED** [review]
LGTM!

**Pull request merged**